deployment/nginx.coffee

```diff
@@ -365,6 +365,17 @@ module.exports.create = (KONFIG, environment)->
         resolver 8.8.8.8;
       }
 
+      # redirect /s3/:bucket/:key to :bucket.s3.amazonaws.com/:key
+      location ~^/s3/([^/]*)/(.*)$ {
+        proxy_pass            "https://$1.s3.amazonaws.com/$2";
```

Comment:
A trivial note. I'd prefer `https://s3.amazon.aws.com/$1/$2` to avoid resolving subdomain name.